data/mods/ssb/statuses.ts

```diff
@@ -38,4 +38,47 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
 	IMPORTANT: Obtain the username from getName
 	*/
 	// Please keep statuses organized alphabetically based on staff member name!
+	perishsong: {
```

Comment:
are you sure this works? I'm pretty sure Perish Song is a name effect already

Reply:
oh yeah forgot that move exists